machines/overview.html.markerb

```diff
@@ -117,6 +117,27 @@ Updating a Machine takes it down (like with `fly machine stop`), applies configu
 not changing the image, so we don't have to go fetch it from the global registry, this is fast, for the same reason `stop` and `start`
 are; we've already done the heavy lifting.
 
+### Static Machine IPs
+
+Static egress IPs can be attached to a machine. These IPs survive a [machine migration].
+
+Static egress IPs are useful when your machine needs to connect to a service that requires allowlisting a specific set of IPs.
+If supported, it is recommended to use [Wireguard].
```

Comment:
Not sure what this means- use Wireguard for what? (also, is `[Wireguard]` supposed to link somewhere)